ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2

```diff
@@ -13,6 +13,7 @@ compute:
       osDisk:
         sizeGB: 120
       vmType: server
+      affinityGroupsNames: []
```

Comment:
I would suggest you can add this explicitly in the deployment code because this parm will not work in the case of OCP 4.7

Reply:
Noted

Reply:
@suchita-g if it will break OCP 4.7 deployment shouldn't we fix this?